src/main/java/Sakura/Deadline.java

```diff
@@ -0,0 +1,46 @@
+package Sakura;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a deadline task.
+ */
+public class Deadline extends Task {
+
+    protected LocalDateTime by;
+
+    /**
+     * Constructor for a deadline.
+     *
+     * @param description description of the task.
+     * @param by The deadline in datetime format for the task.
+     */
+    public Deadline(String description, String by) {
+        super(description);
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HHmm");
+        this.by = LocalDateTime.parse(by, formatter);
+    }
+
+    /**
+     * Convert the deadline into the specific string for data saving.

```

Comment:
Remember to use the correct form.
```suggestion
     * Converts the deadline into the specific string for data saving.
```